nemo/collections/cv/modules/linear_projection_video_front_end.py

```diff
@@ -0,0 +1,104 @@
+import torch
+from torch import nn
+
+from nemo.core.classes.module import NeuralModule
+
+
+class LinearProjectionVideoFrontEnd(NeuralModule):
+
+    """
+    Linear Projection Video Front-End for Lip Reading
+
+    The spatial dimension is flattened and projected to dim_output using a Linear layer.
+    This is equivalent to having a convolution layer with a kernel size of the size of the image.
+    Circle crop can be used as pre-processing to crop the image as a circle around lips and ignore corner pixels
+
+    Args:
+        in_channels: number of inputs video channels, 1 for grayscale and 3 for RGB
+        in_height: image height
+        in_width: image width
+        dim_output: output feature dimension for linear projection
+        out_channels_first: Whether outputs should have channels_first format (Batch, Dout, Time) or channels_last (Batch, Time, Dout)
+        circle_crop: crop the image as a circle before the Linear layer, default to False
+        circle_radius: the circle radius, default to 1 for full circle
+    
+    """
+
+    def __init__(
+        self,
+        in_channels=1,
+        in_height=88,
```

Comment:
Where do these defaults come from? better to not set them and enforce the caller to set them explicitly

Reply:
1x88x88 is a usual video size in lip reading papers, I will not set in_channels, in_height, in_width and dim_output

Reply:
done